src/StackExchange.Redis/ConfigurationOptions.cs

```diff
@@ -804,9 +804,12 @@ internal bool IsAzureEndpoint()
             {
                 if (ep is DnsEndPoint dnsEp)
                 {
-                    if (azureRedisHosts.Any(s => dnsEp.Host.ToLowerInvariant().EndsWith(s)))
+                    foreach (var host in azureRedisHosts)
```

Comment:
nice unroll; just as a "ooh, while we're there, this could be better" - would this work with `dnsEp.Host.EndsWith(host, StringComparison.InvariantCultureIgnoreCase)` ?

Reply:
You got it!